.travis.yml

```diff
@@ -10,9 +10,6 @@ os:
 git:
   depth: 300
 
-before_script:
-  - npm run bootstrap
```

Comment:
Travis automatically runs `npm install` before running any `before_script` entries. There is no need to run `npm install` again!